flocker/node/agents/test/test_blockdevice.py

```diff
@@ -754,7 +754,12 @@ def __init__(self, block_api, live_nodes=()):
         self.live_nodes = live_nodes
 
     def list_live_nodes(self):
-        return [self.compute_instance_id()] + list(self.live_nodes)
+        result = {self.compute_instance_id():
+                  set(unicode(i) for i in get_all_ips()
+                      if i != b"127.0.0.1")}
+        result.update({node: [u"10.1.1.{}".format(i)]
+                       for i, node in enumerate(self.live_nodes)})
```

Comment:
:grey_question: would you mind documenting the type of `self.live_nodes` on line 751? Slowed my review down not having it there.